packages/google_maps_flutter/google_maps_flutter_platform_interface/test/types/bitmap_test.dart

```diff
@@ -21,10 +21,17 @@ void main() {
           BitmapDescriptor.fromJson(json);
 
       expect(descriptorFromJson, isNot(descriptor)); // New instance
-      expect(identical(descriptorFromJson.toJson(), json), isTrue); // Same JSON
+      expect(descriptorFromJson.toJson(), json);
```

Comment:
Since the bitmaps are no longer just wrapping JSON, `toJson` will not return the same reference as was passed to `fromJson`, though they will still be equal by comparison.